Projectile.cs

```diff
@@ -27,26 +27,12 @@ public Vector2 Velocity
             get { return velocity; }
             set { velocity = value; }
         }
-        private Vector2 ScrSize;
-        public Vector2 ScreenSize
-        {
-            get { return ScrSize; }
-            set { ScrSize = ScreenSize; }

```

Comment:
these set functions should say ScrSize = value but i think screensize should probably be read-only anyway, so we only need a get function
this is because after its set with the constructor it wont change